backend/src/agents/__init__.py

```diff
@@ -3,8 +3,26 @@
 from .agent import Agent, agent_metadata, convert_to_mistral_tool
 from .datastore_agent import DatastoreAgent
 from .maths_agent import MathsAgent
-from .goal_achieved_agent import GoalAchievedAgent
-from .unresolvable_task_agent import UnresolvableTaskAgent
+from .validator_agent import ValidatorAgent
 
-__all__ = ["Agent", "DatastoreAgent", "MathsAgent", "GoalAchievedAgent", "UnresolvableTaskAgent", "agent_metadata",
-           "create_tasks", "tool_metadata", "Parameter", "convert_to_mistral_tool"]
+validator_agent = ValidatorAgent()
+
+
+def get_agent_details(agent):
+    return {"name": agent.name, "description": agent.description}
+
+
+agents = [DatastoreAgent(), MathsAgent()]

```

Comment:
Why are some agents listed as agents, but create_tasks for example isn't in there?

Reply:
Create tasks agent doesn't currently use our class / decorators, and this is the list of agents that are available to the supervisor / router, so create tasks probably shouldn't be in this array, unless we want that ability in the future 🤔

Reply:
Aah yes I see, thanks I understand the logic now :)